docs/contributing/contributing.md

```diff
@@ -0,0 +1,122 @@
+    - [Preview Draft PRs on a Local Web Browser](#preview-draft-prs-on-a-local-web-browser)
+  
+
+Thank you for considering making contributions. We appreciate your interest in helping us to create and maintain awesome tutorials.
+
+To set up your environment for success, follow our [technical set up](technical-setup.md) guidelines.
+
+## Using this repo
+
+- To provide feedback, file an issue and provide abundant details to help us understand how we can make it better.
+- To provide feedback and a fix, you can make a direct contribution. This repo is protected since we provide the code and the docs to help you learn. If you're not a member or maintainer, fork the repo and then submit a pull request (PR) from your forked repo to the `develop` branch.
+- Start by creating a draft pull request. Create your draft PR early, even if your work is just beginning or incomplete. Your draft PR indicates to the community that you're working on something and provides a space for conversations early in the development process. Merging is blocked for `Draft` PRs, so they provide a safe place to experiment and invite comments. 
+
+## Reviewing technical content PRs 
+
+Some of the best content contributions come during the PR review cycles. Follow best practices for technical content PR reviews just like you do for code reviews. For in-line suggestions, use the [GitHub suggesting feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request). The PR owner can merge in your suggested commits one at a time or in batch (preferred). When you do a more granular extensive review that results in more than 20 in-line suggestions, go ahead and check out the branch and make the changes yourself. 
+
+## Writing and contributing
+
+We welcome contributions to the tutorials. 
+
+Our technical content follows the [Google developer documentation style guide](https://developers.google.com/style). Highlights to help you get started:
+
+- [Highlights](https://developers.google.com/style/highlights)
+- [Word list](https://developers.google.com/style/word-list)
+- [Style and tone](https://developers.google.com/style/tone)
+- [Writing for a global audience](https://developers.google.com/style/translation)
+- [Cross-references](https://developers.google.com/style/cross-references)
+- [Present tense](https://developers.google.com/style/tense)
+
+The Google guidelines include more material than is listed here and are used as a guide that enables easy decision making about proposed content changes. 
+
+Other useful resources:
+
+- [Google Technical Writing Courses](https://developers.google.com/tech-writing)
+- [GitHub Guides Mastering Markdown](https://guides.github.com/features/mastering-markdown/)
+
+## Folder structure
+/TODO update for actual structure 
+Each tutorial lives in a self-describing folder: `blog`, `hello-world`, `voter`, and so on.
```

Comment:
In Starport we have it slightly different. Each tutorial lives in a self-describing folder, or if it can live in a single file, it does so with a name-of-the-tutorial.md file.

Reply:
Starport developer guide tutorials live in the docs/guide folder